src/feature.js

```diff
@@ -183,27 +201,43 @@ export default class Feature {
    * If it has multiple values, those values will be concatenated with a default separator and
    * returned in a single string. Values composing this string are sorted according
    * to each value's sort order.
+   * NOTE: If object contains a single value and it is a number, it will be converted to a string.
    * @return {string} A single value string.
    */
   get value () {
     return this.values.join(this.constructor.joinSeparator)
   }
 
+  /**
+   * Returns a feature value, if Feature object contains a single value. If no value is stored,
+   * returns `undefined`. If feature has more than one value, throws an error.
+   * This method allows to avoid conversion of a value to the string type as is the case
+   * with other methods.
+   *
+   * @returns {undefined|*} - A single value in a format in which it is stored or `undefined`
+   *          if feature has no value.
+   */
+  get singleValue () {
+    if (this._data.length === 0) return undefined

```

Comment:
I believe you don't need to define explicitly `undefined` here.
It would be undefined anyway.

Reply:
Do you suggest to remove this line? If we do that, then the next line executed will be `return this._data[0].value`. `this._data[0]` will be undefined, and it will throw an error trying to access a `value` prop of undefined. Or are you talking about anything else?

Reply:
I meant to do it this way
```
if (this._data.length === 0) return
```

Reply:
Oh yes, blank return statement makes sense here. Will update